### PR TITLE
feat: update all dependencies and bundlers

### PR DESCRIPTION
**Pull request by brandonocasey**
Updates webpack, browserify, rollup, and babel to more recent versions. With that comes a few code changes that I will comment on.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
```
Comment (brandonocasey) [inline .babelrc:1]: We now pass everything in directly.

File: src-es6/rollup.js [hunk 2]
```diff
@@ -17,14 +16,45 @@ export default {
     }),
     babel({
       babelrc: false,
+      babelHelpers: 'runtime',
       exclude: 'node_modules/**',
       presets: [
-        ['env', {
+        ['@babel/preset-env', {
```
Comment (brandonocasey) [inline src-es6/rollup.js:18]: plugin name changes

File: src-es6/rollup.js [hunk 3]
```diff
@@ -17,14 +16,45 @@ export default {
...
+  }, {
+    name: `rollupCjsTest`,
+    file: 'dist/rollup-es6-cjs.js',
+    format: 'cjs',
+    exports: 'auto'
+  }, {
+    name: `rollupEsTest`,
+    file: 'dist/rollup-es6-es.js',
+    format: 'es',
+    exports: 'auto'
```
Comment (brandonocasey) [inline src-es6/rollup.js:44]: exports is now required, I also added all the formats that rollup supports.

File: src-es5/rollup.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const config = require('../src-es6/rollup.js');
+
+config.input = config.input.replace('es6', 'es5');
```
Comment (brandonocasey) [inline src-es5/rollup.js:4]: use the es6 config but change all outputs and inputs to use `es5`

File: src-es6/webpack.config.js [hunk 5]
```diff
@@ -8,7 +8,17 @@ module.exports = {
   },
   module: {
     rules: [
-      { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" }
+      {
+        test: /\.js$/,
```
Comment (brandonocasey) [inline src-es6/webpack.config.js:12]: slightly different syntax here

File: package.json [hunk 6]
```diff
@@ -6,11 +6,12 @@
   "scripts": {
     "browserify-es5": "browserify src-es5/index.js > dist/browserify-es5.js",
     "webpack-es5": "webpack --config src-es5/webpack.config.js",
-    "browserify-es6": "browserify -t [ babelify ] src-es6/index.js > dist/browserify-es6.js",
+    "rollup-es5": "rollup -c src-es5/rollup.js",
```
Comment (brandonocasey) [inline package.json:10]: rollup es5 now too.

File: package.json [hunk 7]
```diff
@@ -6,11 +6,12 @@
...
+    "browserify-es6": "browserify -t [ babelify --presets [ @babel/preset-env ]] src-es6/index.js > dist/browserify-es6.js",
```
Comment (brandonocasey) [inline package.json:11]: pass in the preset direcly

**Review by gkatsev — APPROVED** [review]

**Pull request merged**